server.go

```diff
@@ -272,10 +290,24 @@ func handlePacket(s *Server, p interface{}) error {
 		if !ok || f.IsDir {
 			return s.sendError(p, syscall.EBADF)
 		}
-
 		info, err := s.driver.Stat(f.Path)
+
 		if err != nil {
-			return s.sendError(p, err)
+			if err != os.ErrNotExist {
```

Comment:
[nit] it feels more natural for this to come after the `err == os.ErrNotExist` block as an `else if err != nil { ...` block